### DISCUSSION TITLE
Recommended Release Management Process/Flow

### DISCUSSION THREAD

**Discussion by kk2211** [saleor/saleor#10460] [General] [answered]
Hi
Just wanted to know if there is any Saleor specific documentation/ recommended flow to set-up different environments for testing, staging and prod. 
Thanks

**Comment by CossackDex** [comment, marked as answer]
Hi @kk2211, 

Currently, there's no specific documentation for that, but that's a good idea to create ones at some point. For now, if you would like to test how we create our environments, you can create a free developer account here https://cloud.saleor.io/signup and create a staging environment for testing purposes. 

Have a great day!
